content/_components/identity-brand-bar.md

```diff
@@ -6,6 +6,6 @@ modifier_css:
   - bgcolor: '#3c3c3c'
     css_class: su-brand-bar--white
 ---
-The Stanford Brand Bar is one of the three key elements in the Stanford Identity system for the web. 
+The Stanford Brand Bar is one of the three key elements in the Stanford Identity system for the web.
 
-[Learn more about aligning your site with the Stanford Identity](https://elegant-poitras-87214a.netlify.com/page/brand-design-elements-brand/)
+[Learn more about aligning your site with the Stanford Identity](https://elegant-poitras-87214a.netlify.app/page/brand-design-elements-brand/)
```

Comment:
Actually, is there a reason why we don't use relative links here, like in _components/composite-main-navigation.md?
```suggestion
[Learn more about aligning your site with the Stanford Identity](/page/brand-design-elements-brand/)
```